.changeset/twelve-days-itch.md

```diff
@@ -0,0 +1,7 @@
+---
+"@ergoplatform/scorex-buffer": major
```

Comment:
Nice file name. :)

Reply:
Haha I didn't notice it, nothing worse than a 12 day itch